src/index.js

```diff
@@ -0,0 +1,58 @@
+import './styles.css';
+import { sortIndex } from './functions';
+import { setToLocalStorage, getFromLocalStorage, reloadToDo } from './storage';
+import { addToDo, editToDo, clearAll } from './addremove.js';
+
+const toDoList = [];
+
+// populate list
+const populate = (toDoList, sort) => {
+  let sortedTodo = [];
+  if (sort) {
+    sortedTodo = toDoList.sort((a, b) => a.index - b.index);
+  } else {
+    sortedTodo = toDoList;
+  }
+
+  for (let i = 0; i < sortedTodo.length; i += 1) {
+    let style = '';
+    let checkbox = '';
+    if (sortedTodo[i].completed) {
+      style = 'text-decoration: line-through;';
+      checkbox = 'checked';
+    } else {
+      style = 'text-decoration: none;';
+      checkbox = '';
+    }
+    // create list item
+    const list = document.createElement('example');
```

Comment:
If you want to import all the elements, this is a bad method because it is redefined every loop so the elements you import will not matter and the function will not work as you want.

**Solution**:
Define it outside of the for loop